README.md

```diff
@@ -136,6 +113,18 @@ rake yoksis:import_departments
 
 Third-party integrations are located under `app/services/foo/v1`. Follow up /docs subfolder (ie. `app/services/foo/v1/docs` for SOAPUI templates.
 
+## Background Jobs
+
+This app heavily uses Sidekiq (as ActiveJob objects) for perfoming background jobs. Make sure to start sidekiq for processing queues:
+
+```
+bundle exec sidekiq -q high -q low
```

Comment:
@huseyin sidekiq'i aktive ettim. şimdilik high ve low olmak üzere iki queue kullanıyorum. deploy'a çıkarken sidekiq'i de eklemek gerekecek (veya bu iki queue için uyandırmak).